### PR TITLE
v0.9.0 release

### PR DESCRIPTION
**Pull request by ppstacy**
update Rdocs/changelogs.rst and causalml/init.py to reflect the changes.

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -2,6 +2,30 @@
...
+------------------
+- CausalML won the 1st prize at the poster session in UberML'20
+- DoWhy integrated CausalML starting v0.4 (`release note <https://github.com/microsoft/dowhy/releases/tag/v0.4>`_)
+- CausalML team welcomes new project leadership, Mert Bay
+- We have 4 new community contributors, Mario Wijaya (`@mwijaya3 <https://github.com/mwijaya3>`_), Harry Zhao (`@deeplaunch <https://github.com/deeplaunch>`_), Christophe (`@ccrndn <https://github.com/ccrndn>`_) and Georg Walther (`@waltherg <https://github.com/waltherg>`_). Thanks for the contribution!
+
+Major Updates
+~~~~~~~~~~~~~
+- Add feature importance and its visualization to UpliftDecisionTrees and UpliftRF by @yungmsh (`#220 <https://github.com/uber/causalml/pull/220>`_)
+- Add feature selection example with Filter methods by @paulluo0106 (`#223 <https://github.com/uber/causalml/pull/223>`_)
```
Comment (paullo0106) [inline docs/changelog.rst:15]: small typo here, it's @paullo0106  : )

same for line 21 and 22

**Review by paullo0106 — CHANGES_REQUESTED** [review]
minor typo items, thanks!

File: docs/changelog.rst [hunk 2]
```diff
@@ -2,6 +2,30 @@
...
+Minor Updates
+~~~~~~~~~~~~~
+- Implement propensity model abstraction for common interface by @waltherg (`#223 <https://github.com/uber/causalml/pull/233>`_)
+- Fix bug in BaseSClassifier and BaseXClassifier by @yungmsh and @ppstacy (`#217 <https://github.com/uber/causalml/pull/217>`_), (`#218 <https://github.com/uber/causalml/pull/218>`_)
+- Fix parentNodeSummary for UpliftDecisionTrees by @paulluo0106 (`#238 <https://github.com/uber/causalml/pull/238>`_)
+- Add pd.Series for propensity score condition check by @paulluo0106 (`#242 <https://github.com/uber/causalml/pull/242>`_)
+- Fix the uplift random forest prediction output by @ppstacy (`#236 <https://github.com/uber/causalml/pull/236>`_)
+- Add functions and methods to init for optimization module by @mwijaya3 (`#228 <https://github.com/uber/causalml/pull/228>`_)
+- Install GitHub Stale App to close inactive issues automatically @jeongyoonlee (`#237 <https://github.com/uber/causalml/pull/237>`_)
+- Update documentation by @deeplaunch, @ccrndn, @ppstacy(`#214 <https://github.com/uber/causalml/pull/214>`_, `#231 <https://github.com/uber/causalml/pull/223>`_, `#232 <https://github.com/uber/causalml/pull/223>`_)
```
Comment (paullo0106) [inline docs/changelog.rst:26]: the links should be `.../pull/231` and `.../pull/232` for #231 and #232, respectively

File: docs/changelog.rst [hunk 3]
```diff
@@ -2,6 +2,30 @@
...
+- We have 4 new community contributors, Mario Wijaya (`@mwijaya3 <https://github.com/mwijaya3>`_), Harry Zhao (`@deeplaunch <https://github.com/deeplaunch>`_), Christophe (`@ccrndn <https://github.com/ccrndn>`_) and Georg Walther (`@waltherg <https://github.com/waltherg>`_). Thanks for the contribution!
+
+Major Updates
+~~~~~~~~~~~~~
+- Add feature importance and its visualization to UpliftDecisionTrees and UpliftRF by @yungmsh (`#220 <https://github.com/uber/causalml/pull/220>`_)
+- Add feature selection example with Filter methods by @paulluo0106 (`#223 <https://github.com/uber/causalml/pull/223>`_)
+
+Minor Updates
+~~~~~~~~~~~~~
+- Implement propensity model abstraction for common interface by @waltherg (`#223 <https://github.com/uber/causalml/pull/233>`_)
```
Comment (paullo0106) [inline docs/changelog.rst:19]: replace #223 with #233

**Review by paullo0106 — CHANGES_REQUESTED** [review]

File: docs/changelog.rst (hunk 1 above)
Comment (ppstacy) [reply docs/changelog.rst:15]: Thanks @paullo0106 ! Sorry about that

**Review by jeongyoonlee — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**